packages/sdk/client-services/src/packlets/services/queue-syncer.ts

```
@@ -0,0 +1,214 @@
+      const encoded = encoder.encode(message);
+      this.#edgeClient.send(
+        createBuf(MessageSchema, {
+          source: {
+            identityKey: this.#edgeClient.identityKey,
+            peerKey: this.#edgeClient.peerKey,
+          },
+          serviceId: this.#getTargetServiceId(message),
+          payload: { value: bufferToArray(encoded) },
+        }),
+      );
+    });
```

Biome (2.3.13)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)